src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+/*Class created by Priyanka Agarwal as per code written
+on codeU directions. Only added package and import statements
+ */
+
+package codeu.chat.common;
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+
+public final class ServerInfo {
+    private final static String SERVER_VERSION = "1.0.0";
+
+    public Uuid version;
+    public ServerInfo() {
```

Comment:
You might think about adding a test class for this class? Even if it's not in the tutorial, it's a good habit to get into. Let me know if you have questions about that. Or I think Matt's activity includes adding a test, so maybe talk to him too?